bot/seasons/easter/egg_decorating.py

```diff
@@ -0,0 +1,95 @@
+import logging
+import random
+from io import BytesIO
+from pathlib import Path
+
+
+import discord
+from PIL import Image
+from discord.ext import commands
+
+log = logging.getLogger(__name__)
+
+COLOURS = [
+    (255, 0, 0, 255), (255, 128, 0, 255), (255, 255, 0, 255), (0, 255, 0, 255),
+    (0, 255, 255, 255), (0, 0, 255, 255), (255, 0, 255, 255), (128, 0, 128, 255)
+]  # Colours to be replaced - Red, Orange, Yellow, Green, Light Blue, Dark Blue, Pink, Purple
+
+IRREPLACEABLE = [
+    (0, 0, 0, 0), (0, 0, 0, 255)
+]  # Colours that are meant to stay the same - Transparent and Black
+
+
+class EggDecorating(commands.Cog):
+    """A Command that lets you decorate some easter eggs!"""
+
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.command(aliases=["decorateegg"])
+    async def eggdecorate(self, ctx, *colours):
+        """
+        This 'paints' a beautiful egg using inputted colours
+
+        It picks from a random set of designs and alters the colours to the user's liking
+        """
+
+        if len(colours) < 2:
+            return await ctx.send("You must include at least 2 colours!")
+
+        invalid = []
+        converted = []
+        for c in colours:
+            try:
+                colour = await commands.ColourConverter().convert(ctx, c)
+                # Attempts to convert the arguments into discord.Colour
```

Comment:
It's probably worth having some more common colors defined as fallbacks for those that discord.py doesn't define (e.g. yellow, pink).

This could also be more or less mitigated by the comment below.

Reply:
Better support for a wider array of colour names would be a good idea, for sure.

Reply:
I've added in two new resources to the PR that can be used:
- HTML colour names and values
- XKCD colour names and values

My review has suggested changes that make use of them.